modules/configuring-firewall.adoc

```diff
@@ -148,6 +143,21 @@ the routes work. If you are the cluster administrator and do not want to allow
 that is specified in the `spec.route.hostname` field of the
 `consoles.operator/cluster` object if the field is not empty.
 
+. Allowlist the following URLs for optional operators:
++
+[cols="3,2,4",options="header"]
+|===
+|URL | Port | Function
+
+|`registry.connect.redhat.com`
+|443, 80
+|Required for all third party certified operators.
```

Comment:
AFAIK, this `registry.connect.redhat.com` is not only for all third party certified operators but all ISV images.